packages/repository-json-schema/package-lock.json

```diff
@@ -4,23 +4,554 @@
+			"integrity": "sha512-8fRE76xNwNttVEF2TwxJDGBLWthUkHWSldmfuBzVRmEDWOtu4XdINTgN7TDWzuLg4bbeIMLvfMFD9we5YcWkRQ==",
+			"dev": true,
+			"requires": {
+				"@babel/code-frame": "^7.5.5",
+				"@babel/generator": "^7.6.2",
+				"@babel/helper-function-name": "^7.1.0",
+				"@babel/helper-split-export-declaration": "^7.4.4",
+				"@babel/parser": "^7.6.2",
+				"@babel/types": "^7.6.0",
+				"debug": "^4.1.0",
+				"globals": "^11.1.0",
+				"lodash": "^4.17.13"
+			}
+		},
+		"@babel/types": {
+			"version": "7.6.1",
+			"resolved": "https://registry.npmjs.org/@babel/types/-/types-7.6.1.tgz",
+			"integrity": "sha512-X7gdiuaCmA0uRjCmRtYJNAVCc/q+5xSgsfKJHqMN4iNLILX39677fJE1O40arPMh0TTtS9ItH67yre6c7k6t0g==",
+			"dev": true,
+			"requires": {
+				"esutils": "^2.0.2",
+				"lodash": "^4.17.13",
+				"to-fast-properties": "^2.0.0"
+			}
+		},
+		"@hapi/address": {
+			"version": "2.1.2",
+			"resolved": "https://registry.npmjs.org/@hapi/address/-/address-2.1.2.tgz",
+			"integrity": "sha512-O4QDrx+JoGKZc6aN64L04vqa7e41tIiLU+OvKdcYaEMP97UttL0f9GIi9/0A4WAMx0uBd6SidDIhktZhgOcN8Q==",
+			"dev": true
+		},
+		"@hapi/formula": {
+			"version": "1.2.0",
+			"resolved": "https://registry.npmjs.org/@hapi/formula/-/formula-1.2.0.tgz",
+			"integrity": "sha512-UFbtbGPjstz0eWHb+ga/GM3Z9EzqKXFWIbSOFURU0A/Gku0Bky4bCk9/h//K2Xr3IrCfjFNhMm4jyZ5dbCewGA==",
+			"dev": true
+		},
+		"@hapi/hoek": {
+			"version": "8.2.4",
+			"resolved": "https://registry.npmjs.org/@hapi/hoek/-/hoek-8.2.4.tgz",
+			"integrity": "sha512-Ze5SDNt325yZvNO7s5C4fXDscjJ6dcqLFXJQ/M7dZRQCewuDj2iDUuBi6jLQt+APbW9RjjVEvLr35FXuOEqjow==",
+			"dev": true
+		},
+		"@hapi/joi": {
+			"version": "16.1.4",
+			"resolved": "https://registry.npmjs.org/@hapi/joi/-/joi-16.1.4.tgz",
+			"integrity": "sha512-m7ctezhxjob+dSpXnCNlgAj6rrEpdSsaWu3GWL3g1AybQCU36mlAo9IwGFJwIxD+oHgdO6mYyviYlaejX+qN6g==",
+			"dev": true,
+			"requires": {
+				"@hapi/address": "^2.1.2",
+				"@hapi/formula": "^1.2.0",
+				"@hapi/hoek": "^8.2.4",
+				"@hapi/pinpoint": "^1.0.2",
+				"@hapi/topo": "^3.1.3"
+			}
+		},
+		"@hapi/pinpoint": {
+			"version": "1.0.2",
+			"resolved": "https://registry.npmjs.org/@hapi/pinpoint/-/pinpoint-1.0.2.tgz",
+			"integrity": "sha512-dtXC/WkZBfC5vxscazuiJ6iq4j9oNx1SHknmIr8hofarpKUZKmlUVYVIhNVzIEgK5Wrc4GMHL5lZtt1uS2flmQ==",
+			"dev": true
+		},
+		"@hapi/shot": {
+			"version": "4.1.2",
+			"resolved": "https://registry.npmjs.org/@hapi/shot/-/shot-4.1.2.tgz",
+			"integrity": "sha512-6LeHLjvsq/bQ0R+fhEyr7mqExRGguNTrxFZf5DyKe3CK6pNabiGgYO4JVFaRrLZ3JyuhkS0fo8iiRE2Ql2oA/A==",
+			"dev": true,
+			"requires": {
+				"@hapi/hoek": "8.x.x",
+				"@hapi/joi": "16.x.x"
+			}
+		},
+		"@hapi/topo": {
+			"version": "3.1.4",
+			"resolved": "https://registry.npmjs.org/@hapi/topo/-/topo-3.1.4.tgz",
+			"integrity": "sha512-aVWQTOI9wBD6zawmOr6f+tdEIxQC8JXfQVLTjgGe8YEStAWGn/GNNVTobKJhbWKveQj2RyYF3oYbO9SC8/eOCA==",
+			"dev": true,
+			"requires": {
+				"@hapi/hoek": "8.x.x"
+			}
+		},
+		"@loopback/build": {
```

Comment:
Please revert changes in `package-lock.json` file. Now that you are not adding new dependencies, there is no need to change the lock file either.

For future reference: https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#working-with-dependencies
Here is the important part: Dependencies resolved locally within the monorepo must be excluded from package-lock files.

Reply:
Nice, this problem was catched by our linter, I forget we have one in place!

See https://travis-ci.com/strongloop/loopback-next/jobs/238791798

```
Invalid package-lock entries found!
  packages/repository-json-schema/package-lock.json
    -> @loopback/build
    -> @loopback/context
    -> @loopback/core
    -> @loopback/eslint-config
    -> @loopback/metadata
    -> @loopback/repository
    -> @loopback/testlab

Run the following command to fix the problems:

  $ npm run update-package-locks
```

Please don't run `update-package-locks` and just revert changes in this file. We prefer to update lock files via https://renovatebot.com

Reply:
Sorry, I followed the suggestion of travis.  Reverted.